test/parallel/test-event-emitter-listeners-side-effects.js

```diff
@@ -21,7 +21,6 @@ assert(fl.length === 1);
 assert(fl[0] === assert.fail);
 
 e.listeners('bar');
-assert(!e._events.hasOwnProperty('bar'));
```

Comment:
There is no `_events.hasOwnProperty` anymore – you could do something like `Object.prototype.hasOwnProperty.call(e._events, 'bar')` here, I guess?